### ISSUE TITLE
building EchoXcode example

### ISSUE THREAD

**Issue by mark1010** [grpc/grpc-swift#273]
I'm trying to build EchoXcode following the steps in the doc, with the current head and Xcode Version 9.4.1 (9F2000). 

The project seems to be missing dependencies on czlib, I apologise for the newbie question.

This is the process I've followed:

```
git clone https://github.com/grpc/grpc-swift
cd grpc-swift/

cd third_party
sh RUNME.sh
```
y (return), (return), all installs ok
```
cd ..
make && make project
```
All built ok, (An Echo target appears to have complied in this phase).

Open ./Examples/EchoXcode/Echo. xcodeprj in Xcode
Left Hand Nav (LHN) - right click Echo project Add Files, SwiftGRPC.xcodeproject, from parent folder.

Active Scheme is Echo (Echo project) > My Mac

Initial build fails, missing CgRPC.

LHN - click Echo project, General tab opens.

(Signing, Change Team).

Linked Frameworks and Libraries, 4x greyed out (unresolved) entries, only SwiftProtobuf.framework is 
listed.

base on README.md and `Echo.xcodeproj/project.pbxproj`, `PBXBuildFile section` the missing frameworks are: BoringSSL.framework,  Czlib.framework, CgRPC.framework, SwiftGRPC.framework.

+ button allows addition of: BoringSSL, CgRPC, Commander, SwiftGRPC, SwiftProtobufPluginLibrary, and is missing Czlib.

Add BoringSSL, SwiftGRPC, CgRPC.

Build with Command-B

Build fails, no changes,  missing CgRPC.
<img width="269" alt="screen shot 2018-07-20 at 13 51 24" src="https://user-images.githubusercontent.com/10847477/43003324-1812000e-8c24-11e8-9c37-809c567e0a3c.png">

Under Build Phases, Target Dependencies:
Add: BoringSSL, CgRPC, SwiftGRPC

no change to errors.

It's not clear to me whether I'm supposed to have provide Czlib, or is that part of the package management(?), and / or whether it's: Build Phases, Target Dependencies or Linked Frameworks / Libs that's causing the CgRPC issue, as CgRPC is there.

**Comment by MrMage** [comment]
Hi Mark, thanks for the report! I think `Czlib` is an old module that is no longer module. Could you please try simply removing the module from the Xcode project and see if building works then? If not, could you try adding "libz" instead to the target's dependencies? And if you get that to work, would you mind filing a PR with those changes so we can include them in the repo? :-)

**Comment by MrMage** [comment]
@mark1010 is this still affecting you?

**Comment by mark1010** [comment]
@MrMage Echo is now working for me thanks.

 A few notes:

**Echo**
I didn't follow `/grpc-swift/Examples/EchoXcode/README.md` to the letter.

The process I followed was:
1. `make && make project`
2. Open Echo.xcodeproj
3. Add files, SwiftGRPC.xcodeproj
4. Add SwiftGRPC.framework under Linked Frameworks and Libraries
5. Build / run

Step 4 is an addition to the README, and I didn't need `gRPC, CgRPC, BoringSSL, and SwiftProtobuf` to build / run.

**SwiftGRPC.xcodeproj** 
I noticed that SwiftGRPC.xcodeproj has Bundle Identifier: Boring SSL

**Examples**
`/grpc-swift/Examples/README.md` refers to the 'third_party' folder / process, which I don't think you have now with the new build process.

Mark
